### PR TITLE
Sidebar with Categories, Search Feature

### PR DESCRIPTION
**Pull request by sam-shervin**

### CODE REVIEW

File: .DS_Store
Comment (AbhijithGanesh) [inline .DS_Store:1]: Delete this file pleasee

File: css/searchbar.css
Comment (AbhijithGanesh) [inline css/searchbar.css:1]: Can this be implemented using Tailwindcss?

File: css/sidebar.css
Comment (AbhijithGanesh) [inline css/sidebar.css:1]: Same as searchbar.css, I think this should be implemented using tailwind as well

File: gatsby-node.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const createPages: GatsbyNode["createPages"] = async ({
       component: markDownTemplate,
       context: {
         author: node.frontmatter.author,
-        linkedin = node.frontmatter.linkedin,
+        linkedin: node.frontmatter.linkedin,
```
Comment (AbhijithGanesh) [inline gatsby-node.ts:36]: Good Catch

File: src/.DS_Store
Comment (AbhijithGanesh) [inline src/.DS_Store:1]: Delete this file

File: src/components/.DS_Store
Comment (AbhijithGanesh) [inline src/components/.DS_Store:1]: Delete this file

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

File: tailwind.config.js [hunk 2]
```diff
@@ -40,6 +40,11 @@ module.exports = {
           green: "#00C180",
           logo: "#31CE9F",
         },
+        blur_opacity: {
+          one: 'rgba(255, 255, 255, 0.8)',
+          two: 'rgba(255, 255, 255, 0.6)',
+          three: 'rgba(255, 255, 255, 0.4)',
+        }
```
Comment (AbhijithGanesh) [inline tailwind.config.js:47]: I dont think this is the way to do it

File: src/content/working-with-neural-networks.mdx [hunk 3]
```diff
@@ -28,7 +29,7 @@ have dominated in machine intelligence
 - Seaborn([seaborn: statistical data visualization — seaborn 0.12.1 documentation (pydata.org)](https://seaborn.pydata.org/))
 - Tensorflow([https://www.tensorflow.org/](https://www.tensorflow.org/))
 - Keras([https://keras.io/](https://keras.io/))
-- If one has a strong enough pc(with at least 16 GB RAM, M.2 SSD, RTX2080 super , i7 11th gen or ryzen 7 5000 series) they can do it on jupyter([https://jupyter.org/](https://jupyter.org/)) else use keggle([https://www.kaggle.com/](https://www.kaggle.com/))
+- If one has a strong enough pc(with at least 16 GB RAM, M.2 SSD, RTX2080 super , i7 11th gen or ryzen 7 5000 series) they can do it on jupyter([https://jupyter.org/](https://jupyter.org/)) else use kaggle([https://www.kaggle.com/](https://www.kaggle.com/))
```
Comment (AbhijithGanesh) [inline src/content/working-with-neural-networks.mdx:32]: Good catch

File: src/content/.DS_Store
Comment (AbhijithGanesh) [inline src/content/.DS_Store:1]: Delete this file

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

File: css/searchbar.css
Comment (sam-shervin) [reply css/searchbar.css:1]: except the scrolling features, yes

File: css/sidebar.css
Comment (sam-shervin) [reply css/sidebar.css:1]: done

File: tailwind.config.js (hunk 2 above)
Comment (sam-shervin) [reply tailwind.config.js]: removed that part! Thank you for letting me know, I thought I discarded it earlier!

**Comment by sam-shervin** [comment]
All requested changes are resolved!

**Review by AbhijithGanesh — APPROVED** [review]

**Pull request merged**